x-pack/test/fleet_api_integration/apis/epm/managed_packages.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+import { skipIfNoDockerRegistry } from '../../helpers';
+import { setupFleetAndAgents } from '../agents/services';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  const oldPackageKey = 'apm-0.1.0';
```

Comment:
This was my fallback plan since I couldn't get the jest integration test suite above to run. However, it seems we actually hydrate the test registry with real packages, so `apm` here pulls `apm-0.4.0` from the actual registry and makes it hard to test. 

I don't think we can use the API integration tests because we can't actually mock our hardcoded lists of `default` or `auto update` packages since they're built into the Kibana artifact.

Reply:
If need be, we could also add this to the manual test plan instead for now. I think this should be fairly easy to do manually, though it does require going through the Kibana rollback hoops with taking and restoring snapshots.